src/panels/lovelace/custom-card-helpers.ts

```diff
@@ -0,0 +1,11 @@
+import { createRowElement as createRowElement_ } from "./create-element/create-row-element";
+import { createCardElement as createCardElement_ } from "./create-element/create-card-element";
+import { createBadgeElement as createBadgeElement_ } from "./create-element/create-badge-element";
+import { createHeaderFooterElement as createHeaderFooterElement_ } from "./create-element/create-header-footer-element";
+import { createHuiElement as createHuiElement_ } from "./create-element/create-hui-element";
```

Comment:
You can just use export without the `import as` -> `export`
```suggestion
export { createHuiElement } from "./create-element/create-hui-element";
```